test/views/vat/RegisteredForVATECSalesViewSpec.scala

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright 2018 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package views.vat
+
+import forms.vat.RegisteredForVATFormProvider
+import models.vat.RegisteredForVAT
+import play.api.data.Form
+import play.twirl.api.HtmlFormat
+import viewmodels.ViewAction
+import views.behaviours.ViewBehaviours
+import views.html.vat.registeredForVAT
+
+class RegisteredForVATECSalesViewSpec extends ViewBehaviours {
+
+  val messageKeyPrefix = "registeredForVATECSales"
+
+  val form = new RegisteredForVATFormProvider()()
+  lazy val viewAction = ViewAction(controllers.routes.IndexController.onPageLoad(), "")
+
+  val serviceInfoContent = HtmlFormat.empty
+
+  def createView =
+    () => registeredForVAT(frontendAppConfig, form, viewAction)(serviceInfoContent)(fakeRequest, messages)
+
+  def createViewUsingForm =
+    (form: Form[_]) => registeredForVAT(frontendAppConfig, form, viewAction)(serviceInfoContent)(fakeRequest, messages)
+
+  "RegisteredForVATECSales view" must {
+    behave like normalPage(createView, messageKeyPrefix)
```

Comment:
add content test

Reply:
The only content on the page is the header and radio buttons, both have tests